backend/src/apiserver/storage/experiment_store.go

```diff
@@ -60,11 +62,22 @@ func (s *ExperimentStore) ListExperiments(filterContext *model.FilterContext, op
 	errorF := func(err error) ([]*model.Experiment, int, string, error) {
 		return nil, 0, "", util.NewInternalServerError(err, "Failed to list experiments: %v", err)
 	}
+	q := s.dbDialect.QuoteIdentifier
+	opts.SetQuote(s.dbDialect.QuoteIdentifier)
+
+	// Fix for wrong sort key prefix.
+	opts.SetSortByFieldPrefix(q("experiments") + ".")
+	opts.SetKeyFieldPrefix(q("experiments") + ".")
```

Comment:
this feels like an anti-pattern, since we're overriding the listable's fieldprefix retrieved via `GetSortByFieldPrefix()` or `GetKeyFieldPrefix()` 

why did we need to do this for `experiments` but not `pipelines`? wouldn't the same issue arise for pipeline and other listable models? 

if so, maybe we can just update the return values for the model structs (e.g. pipeline, experiment, etc.) to return the prefix without `.`, something like: 

```
func (e *Experiment) GetSortByFieldPrefix(name string) string {
	return "experiments"
}
```

and then have `list.go` handle adding the quotes on values retrieved via `GetSortByFieldPrefix/GetKeyFieldPrefix`, what do you think?